src/style.css

```
@@ -1,3 +1,74 @@
-body {
+:root {
+  --active-color: #00acdc;
+}
+
+* {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  background: #fff;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  flex-shrink: 0;
+}
+
+.logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+  align-items: center;
+}
+
+.logo__img {
+  flex-shrink: 0;
+}
+
+.nav__list {
+  display: flex;
+  margin-right: 0;
+  padding: 0;
+  list-style: none;
+
+}
+
+.nav__link {
+  display: block;
+  height: 60px;
+  line-height: 60px;
+  position: relative;
+  color: #000;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  text-transform: uppercase;
+  text-decoration: none;
+  cursor: pointer;
+}
+
+.nav__list li:not(:last-child) {
+  margin-right: 20px;
+}
```

Do not rise selector specificity without necessary
```suggestion
.nav__list:not(:last-child) {
  margin-right: 20px;
}
```